emcc.py

```diff
@@ -3775,15 +3775,14 @@ def modularize():
 
   src = '''
 %(maybe_async)sfunction(config) {
-  var %(EXPORT_NAME)s = config || {};
+  var Module = config || {};
```

Comment:
Is `config` used anywhere else? If not, I suggest to change the function signature to just

`function(Module={})`

I think this suggestion was mentioned in the Safari issue, but for some reason not implemented.

Reply:
The safari issue was that this is not supported there:

```
function(Module) {
  var Module = Module || {};
```

But, as you say, we could do `function(Module={})`.  However, I don't think we use default argument anywhere else in emscripten yet, so maybe that would be best don't as a followup? It does looks like default argument are widely supported:  https://caniuse.com/?search=default%20parameters

Reply:
> The safari issue was that this is not supported there:

Oh... I must've mixed up the reasons. Typical case where a comment on the line would help :P

> It does looks like default argument are widely supported

Yeah it was part of ES6. Since we already use other ES6 features, there's no harm in using default params as well.

Reply:
We still support compiling to ES5 though, and since rely on closure to transpile (and I'm still working on that part) we only support certain features that closure can transpile without the help of its extra polyfil code.   Sadly we can't we still can't use all ES6 features, but this seems like one that would be easy to closure to transpile so I think it should be fine.

I'd think I'd like to land this first, then try to switch to a default argument as a followup.